src/eval-frame/initialize-user-variables.js

```diff
@@ -1,5 +1,6 @@
 import { updateUserVariables } from "./actions/actions";
+import { sendActionToEditor } from "./actions/editor-message-senders";
 
-export default store => {
-  store.dispatch(updateUserVariables());
+export default () => {
+  sendActionToEditor(updateUserVariables());
```

Comment:
no longer depends on store or dispatches to store, just sends the editor a action with the list of variables